en-US/win10/samples/arduino-wiring/DCMotor.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: default
+title: Arduino Wiring DC Motor Sample
+permalink: /en-US/win10/samples/arduino-wiring/DCMotor.htm
+lang: en-US
+---
+
+#Arduino Wiring DC Motor Sample
+
+DC motors are some of the basic elements of a maker's toolbox. Whether you're driving autonomous robots or spinning a cooling fan, a DC motor is a relatively simple way to accomplish your task! In this sample, we'll use pulse-width modulation (PWM) to vary the speed of a motor.
+
+##Components
+
+You will need:
+
+* 1 Windows 10 IoT Core enabled device, such as Raspberry Pi 2 or Minnowboard Max.
+* 1 [PCA9685 16-channel 12-bit PWM controller from Adafruit](http://www.adafruit.com/product/815).
+* 1 330 Ω resistor.
+* 1 1N1407 diode.
+* 1 P2N2 222A transistor (NPN)
+* Assorted wires
+
+##Hardware Set Up
+
+The pins on the Raspberry Pi 2 do not provide enough power on their own to run motors and other power-hungry parts. Because the motor needs more current, we'll use a transistor to connect the motor to an external power source. Transistors are like switches; when we give it a small amount of current, it can enable the flow of a much larger current. For our motor, we used a 5v DC adapter that supplies at least 1A of current. It is OK to use an adapter with more current capacity than your motor needs; as the adapter should only supply what is needed by the motor.
```

Comment:
The phrasing "do not provide enough power", likely should be changed to "Motors draw more current than the pins can safely provide"